### DISCUSSION TITLE
Invitation from @CometaHub to participate in an Algorand Builders Talk episode about DAOs

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#164] [General]
@CometaHub has reached out to us on Twitter regarding participation in an upcoming episode of their new "Algorand Builders Talk" series.  The invitation is for an episode related to DAOs.  Here is the correspondence from Twitter:

@CometaHub
> Helloooo team how are you ? 🙂

> I am reaching out to you today because recently we launch a new serie of Twitter Spaces called Algorand Builders Talks

> In this serie we’d like to invite important builders of the ecosystem and chose a topic for every episode. 

> We recorded the 1st episode with the [@deflyapp](https://twitter.com/deflyapp/) and [@xbacked](https://twitter.com/xbacked/) teams and we had a good success !

For reference: a recording of this episode is located [here](https://twitter.com/i/spaces/1ypKdEgaaNrGW).

> And we would love to organize the second episode about DAOs 🔥 and of course having someone from the ASA stats team would be amazing ! 

> Would you be interested to join as guest ? I’d like to record the episode arround Thursday next week.

@asastatscom
> Hi there!  Sorry for the late response!  This sounds like a very interesting proposition, so I've opened a discussion about it on our community Github discussion board, and will announce it on our Discord to understand who could join!

Edit:
One additional message from @asastatscom
> As a quick followup (will also post in the discussion, so please feel free to answer there as well): do you already have additional panel participants scheduled?  If so, are there details you could share?

**Comment by ipaleka** [comment]
The ASA Stats Token Utilization document's initial draft will be published today or tomorrow and it is the very last requirement for opening our marketing campaign designing discussion.

I guess our participation in panels/podcasts like this one, or in a similar Twitter Space that Choice Coin [invited us to join](https://discord.com/channels/906917846754418770/914540797054025788/969982109206859846), should be an outcome of that marketing designing discussion.

We haven't participated so far in a similar event because we haven't shaped what should be our points to emphasize, respectively what stances we take. It is expected that every prominent ASA Stats DAO governor or even future governor - a current member of the ASA Stats community - will be able to participate in such a panel representing the ASA Stats project in an optimal way.

P.S. to clarify more, I should say that by "every prominent ASA Stats DAO governor or even future governor - a current member of the ASA Stats community" I mean people we used to call [Taskers](https://github.com/asastats/docs/blob/main/code-of-conduct.md)... :)

**Reply by AlgoRhythMatic** [reply]
> because we haven't shaped what should be our points to emphasize, respectively what stances we take.

I had a similar thought/concern.  I know what MY thoughts and experiences are in regards to DAOs and our ASA Stats journey to date - but we have not really fully realized "marketing" talking points yet.  I've engaged a bit more with @CometaHub to ask about panel size and other attendees to try and get an idea of what the conversation could be about.  Since this will be recorded and distributed, I want to try and ensure that ASA Stats is well represented!
